h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -2822,4 +2822,6 @@ def cannaryHDFSTest(hdfs_name_node, file_name):
             return False
 
 
-
+def assert_is_type(expected_return_type, actual_return_type):
```

Comment:
I did not know about type checks.  It is well returned.  Deleted mine and replaced with your suggestion.  Thanks.